<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianBlurStepDialog</class>
 <widget class="QDialog" name="GaussianBlurStepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GaussianBlurStepDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Replica-Bold</family>
        <pointsize>15</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Kernelsize</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="kernelSizeSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="kernelSizeNUmber">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="intValue" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Replica-Bold</family>
        <pointsize>15</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>sigma1</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="sigma1Slider">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="sigma1Number">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Replica-Bold</family>
        <pointsize>15</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>sigma2</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="sigma2Slider">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="sigma2Number">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>kernelSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>kernelSizeNUmber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigma1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigma1Number</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sigma2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sigma2Number</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
